### PR TITLE
iOS target unit test

### PR DESCRIPTION
**Pull request by MarekMacko**

### CODE REVIEW

File: shared/src/iosTest/kotlin/com/intive/picover/shared/Test.ios.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+package com.intive.picover.shared
+
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class IosGreetingTest {
+
+	@Test
+	fun testExample() {
+		assertTrue(Greeting().greet().contains("Hello"), "Check 'Hello' is mentioned")
```
Comment (MarekMacko) [inline shared/src/iosTest/kotlin/com/intive/picover/shared/Test.ios.kt:12]: Produced by template. To be removed later when we add at least iOS specific unit test

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -5,10 +5,10 @@ on:
     branches:
       - dev
 jobs:
-  test:
+  android:
     strategy:
       matrix:
-        task: ["app:koverHtmlReportDebug", "ktlint-ruleset:test", "lint:test"]
+        task: [ "app:koverHtmlReportDebug", "ktlint-ruleset:test", "lint:test" ]
```
Comment (MarekMacko) [inline .github/workflows/tests.yml:11]: auto format

File: shared/build.gradle.kts [hunk 3]
```diff
@@ -27,9 +27,6 @@ kotlin {
 	sourceSets {
 		commonMain.dependencies {
 			implementation(libs.coroutines)
-			implementation(libs.firebase.auth)
-			implementation(libs.firebase.firestore)
-			implementation(libs.firebase.storage)
```
Comment (MarekMacko) [inline shared/build.gradle.kts:32]: The strategy here is to have at least one running job on CI, which runs anything against iOS target. Some additional configuration is needed to run `iosX64Test` so I decided to move firebase related files to android sourceset temporarily.

File: .github/workflows/tests.yml [hunk 4]
```diff
@@ -19,3 +19,12 @@ jobs:
...
       - run: ./gradlew ${{ matrix.task }}
+  ios:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'zulu'
+          java-version: 17
+      - run: ./gradlew shared:iosX64Test
```
Comment (MarekMacko) [inline .github/workflows/tests.yml:30]: Failing test `IosGreetingTest` results in failed job, so the initial setup seems to be enough

**Review by gmatyszczak — APPROVED** [review]

**Pull request merged**